style.css

```diff
@@ -0,0 +1,79 @@
+* {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+}
+
+@media screen and (max-width: 767px) {
+  html {
+    box-sizing: border-box;
+  }
+
+  body {
+    background-color: #fff;
+  }
+
+  .header-nav {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    margin: 58px 5%;
+  }
+
+  .logo {
+    color: #6070ff;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 20px;
+  }
+
+  .hamburger {
+    display: block;
+  }
+
+  .heading {
+    background-image: url("./assets/images/mobile-bg-image.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
```

Comment:
- [ ] Good work replicating the original design but there are a few aspects you need to improve on.
     - The headline section that contains your name needs to be 
      adjusted especially with the font size to match the original design
     - Kindly position your background images correctly so that they don't appear to be colliding with other elements of your page 
     
![image](https://user-images.githubusercontent.com/98453979/219611303-885ea05e-95f4-4d40-829b-59c155c3b518.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/219611504-558a89ec-1ced-4f9c-bee1-1483164314a9.png)